### DISCUSSION TITLE
Size of uploaded file exceeds 300MB

### DISCUSSION THREAD

**Discussion by zoyahammad** [vercel/community#6193] [Help]
### Summary

I am trying to deploy a basic python app which uses the Deepface.analyze function. When trying to deploy the app on Vercel, I get this error:

```
Size of uploaded file exceeds 300MB
```
Is it because of large libraries like deepFace and tensorflow? Or could it because of my code structure? 

I have a basic python flask app with this structure:
```
static
--styles.css
templates
--index.html
app.py
requirements.txt
```

### Example

https://github.com/zoyahammad/Cheer-Check

### Steps to Reproduce

https://github.com/zoyahammad/Cheer-Check

**Comment by amyegan** [comment]
Hey @zoyahammad. That can happen when a function is built that exceeds the maximum file size limit. It may be due to large libraries, as you suspect. You can reduce the function size by removing unneeded packages, changing to smaller versions of needed packages, or configuring `includeFiles` and `beforeFiles` in your project's `vercel.json` file. 

[vercel.com/docs/functions/runtimes#bundle-size-limits](https://vercel.com/docs/functions/runtimes#bundle-size-limits)
